sceptre/hooks/cmd.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+from sceptre.hooks import Hook
+from sceptre.exceptions import InvalidHookArgumentTypeError
+
+
+class Cmd(Hook):
+    """
+    Cmd implements a Sceptre hook which can run arbitrary commands.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(Cmd, self).__init__(*args, **kwargs)
+
+    def run(self):
+        """
+        Runs the argument string in a subprocess.
+
+        :raises: sceptre.exceptions.InvalidTaskArgumentTypeException
+        :raises: subprocess.CalledProcessError
+        """
+        try:
+            subprocess.check_call(self.argument, shell=True)
```

Comment:
Ahah sorry - meant "why did we". So what about build servers? Someone could push and execute potentially malicious code. If it were `shell=False`, do we still have the ability to invoke a shell via `/bin/bash -c`?